Public/Src/FrontEnd/MsBuild/PipConstructor.cs

```diff
@@ -715,8 +715,11 @@ private AbsolutePath GetLogDirectory(ProjectWithPredictions projectFile, Qualifi
             var success = Root.TryGetRelative(PathTable, projectFile.FullPath, out var inFolderPathFromEnlistmentRoot);
             Contract.Assert(success);
 
-            // We hardcode the log to go under Logs (and follow the project structure underneath)
-            var result = m_frontEndHost.Configuration.Logging.LogsDirectory
+            // We hardcode the log to go under the output directory Logs/MSBuild (and follow the project structure underneath)
+            // The 'official' log directory (defined by Configuration.Logging) is not stable in CloudBuild across machines, and therefore it would

```

Comment:
>CloudBuild [](start = 96, length = 10)

@ImNarasa was trying to get this working for BuildXL too as he had problems iwth graph cache hits due to this...